eclair-core/src/test/scala/fr/acinq/eclair/channel/states/e/NormalStateSpec.scala

```diff
@@ -1102,6 +1115,43 @@ class NormalStateSpec extends TestkitBaseClass with StateTestsHelperMethods {
     assert(forward.htlc === htlc)
   }
 
+  test("recv RevokeAndAck (one htlc sent, static_remotekey)", Tag("static_remotekey")) { f =>
```

Comment:
I think it would be helpful to run more tests of this suite with `static_remotekey`.
I think at least `recv CMD_FULFILL_HTLC`, `recv UpdateFulfillHtlc`, `recv CMD_FAIL_HTLC` and `recv UpdateFailHtlc` would be useful.

A simple way to do this would be to extract the test content to a function that takes the fixtures as input (for example, `def testReceiveCmdFulfillHtlc(f: FixtureParam): Unit`. Then the existing test simply calls this function, and we can create a duplicate of that test that calls the same function but adds the static_remotekey tag.

Reply:
Good point! I factored out the test body into a function (declared just above the actual tests), now we repeat `CMD_FULFILL_HTLC`, `UpdateFulfillHtlc`, `CMD_FAIL_HTLC` and `UpdateFailHtlc` using `option_static_remotekey` too.